### ISSUE TITLE
How to print with JAX when debug

### ISSUE THREAD

**Issue by whalien-902** [Wanggcong/SparseNeRF#16]

**Comment by Wanggcong** [comment]
Yep, it it a little different to debug JAX code. You might check the offical JAX document. From my side, I only print the loss in train_xxx.py code and dataset_depth_xxx.py.

**Comment by yuannj** [comment]
and how do you use jax to print values similar to the following variables?

Traced<ShapedArray(float32[2])>with<DynamicJaxprTrace(level=0/1)>

**Comment by Wanggcong** [comment]
I do not print some variables. You might refer to Jax's official document for details.

**Comment by Wanggcong** [comment]
Please check the [JAX official website](https://jax.readthedocs.io/en/latest/) for more details.
